src/klein/_headers.py

```diff
@@ -96,44 +92,43 @@ def normalizeRawHeaders(headerPairs):
         yield (normalizeHeaderName(name), value)
 
 
-def normalizeRawHeadersFrozen(headerPairs):
-    # type: (Iterable[Iterable[bytes]]) -> RawHeaders
+def normalizeRawHeadersFrozen(
+    headerPairs: Iterable[Iterable[bytes]],
+) -> RawHeaders:
     return tuple(normalizeRawHeaders(headerPairs))
 
 
-def normalizeRawHeadersMutable(headerPairs):
-    # type: (Iterable[Iterable[bytes]]) -> MutableRawHeaders
+def normalizeRawHeadersMutable(
+    headerPairs: Iterable[Iterable[bytes]],
+) -> MutableRawHeaders:
     return list(normalizeRawHeaders(headerPairs))
 
 
-def getFromRawHeaders(rawHeaders, name):
-    # type: (RawHeaders, AnyStr) -> Iterable[AnyStr]
+def getFromRawHeaders(rawHeaders: RawHeaders, name: AnyStr) -> Iterable[AnyStr]:
     """
     Get a value from raw headers.
     """
     if isinstance(name, bytes):
         name = normalizeHeaderName(name)
         return (v for n, v in rawHeaders if name == n)
 
-    if isinstance(name, Text):
+    if isinstance(name, str):
         rawName = headerNameAsBytes(normalizeHeaderName(name))
         return (headerValueAsText(v) for n, v in rawHeaders if rawName == n)
 
     raise TypeError("name {!r} must be text or bytes".format(name))
 
 
-def rawHeaderName(name):
-    # type: (String) -> bytes
+def rawHeaderName(name: String) -> bytes:
     if isinstance(name, bytes):
         return name
-    elif isinstance(name, Text):
+    elif isinstance(name, str):
         return headerNameAsBytes(name)
     else:
         raise TypeError("name {!r} must be text or bytes".format(name))
```

Comment:
Should this TypeError be `str or bytes`?

Reply:
yes